msteams-platform/task-modules-and-cards/task-modules/invoking-task-modules.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Invoke and dismiss task modules
+description: Invoke and dismiss task modules.
+author: surbhigupta12
+ms.topic: conceptual
+localization_priority: Normal
+---
+
+# Invoke and dismiss task modules
+
+Task modules can be invoked from tabs, bots, or deep links and what appears in one can be either HTML or an Adaptive Card. There is a lot of flexibility in terms of how task modules are invoked and how to deal with the result of a user's interaction. The following table summarizes how this works:
+
+| Invoked using | Task module is HTML or JavaScript | Task module is Adaptive Card |
+| --- | --- | --- |
+| JavaScript in a tab | * Use the Teams client SDK function `tasks.startTask()` with an optional `submitHandler(err, result)` callback function. <br/><br/> * In the task module code, when the user has performed the actions, call the Teams SDK function `tasks.submitTask()` with a `result` object as a parameter. If a `submitHandler` callback was specified in `tasks.startTask()`, Teams calls it with `result` as a parameter.<br/><br/> * If there was an error when invoking `tasks.startTask()`, the `submitHandler` function is called with an `err` string instead. <br/><br/> * You can also specify a `completionBotId` when calling `teams.startTask()`. Then the `result` is sent to the bot instead. | * Call the Teams client SDK function `tasks.startTask()` with a [TaskInfo object](#the-taskinfo-object) and `TaskInfo.card` containing the JSON for the Adaptive Card to show in the task module pop-up. <br/><br/> * If a `submitHandler` callback was specified in `tasks.startTask()`, Teams calls it with an `err` string, if there was an error when invoking `tasks.startTask()` or if the user closes the task module pop-up using the X at the upper right. <br/><br/> * If the user presses an `Action.Submit` button then its `data` object is returned as the value of `result`. |
```

Comment:
Not clear what this table is talking about? Are these tasks that you can perform? If these are mandatory tasks, need to be numbered procedures.
Also, need more information here. For example, the first step:
"Use the Teams client SDK function tasks.startTask() with an optional submitHandler(err, result) callback function."
How to use this? Is it explained anywhere else? If so, give link here. If not, additional information is required.

Same comment for all the bullet points in this table.

Reply:
Awaiting inputs from Wajeed